04-vector-raster-integration-advanced.Rmd

```diff
@@ -292,13 +298,16 @@ plot(plot.locationSp_HARV,
 
 ```
 
-Looking at the plot we created in the challenge we see all the vegetation plot
-locations (blue) on the Canopy Height Model raster except for one.  Why? Refer
-back to our first figure in this lesson showing the relative extents of all the 
-spatial objects.  Notice that the plot extent (black) extends slightly farther 
-west than the CHM raster (green) extent.  Even though we assigned the extents 
-to match, there isn't any raster data that can appear in that area so it appears
-as white.  
+In the plot above, created in the challenge, all the vegetation plot locations
+(blue) appear on the Canopy Height Model raster layer except for one. One is
+situated on the white space. Why? 
+
+Look back at the first figure in this lesson showing the relative extents of
+all the spatial objects. Notice that the veg. plot extent (black) extends
+slightly farther west than the CHM raster (green) extent. We cropped the raster
+to the vector extent, but there isn't any raster data in that western region of
+our plot area so the vegetation plot in that area appears outside the extent of
+the raster data.
 
```

Comment:
PLEASE CHANGE TO: 
***
Look back at the first figure in this lesson showing the relative extents of all the spatial objects. Notice that the extent for our vegetation plot layer (black) extends further west than the extent of our CHM raster (green). The crop function will make a raster extent **smaller**, it will not expand the extent in areas where there are no data. Thus, extent of our vegetation plot layer will still extend further west than the extent of our (cropped) raster data. 

When we plot the data, `R` will use the extent of the first layer plotted (in this case the CHM raster) as it's plot extent. Thus features in layers that fall outside of the extent of the first layer plotted (the CHM) will not be visible in the plot. 
***
@mjones01 -- what do you think about drawing the two extents here as a plot to show the veg plot extent vs the cropped raster? Then showing the plot with the extent modified (further out) ? thoughts?  Would that help clarify? this seems to be a sticking point.